test/e2e/manifest.yaml

```diff
@@ -22,4 +22,6 @@ DriverInfo:
     controllerExpansion: true
     nodeExpansion: true
     snapshotDataSource: false
+    multipods: false
```

Comment:
Why are we disabling multipod capability here?  vSphere CSI driver even when deployed in single toplogy should support running multiple pods from same volume.